LoopKit/Service/RemoteDataService.swift

```diff
@@ -118,5 +125,9 @@ public extension RemoteDataService {
     func uploadSettingsData(_ stored: [StoredSettings], completion: @escaping (_ result: Result<Bool, Error>) -> Void) {
         completion(.success(false))
     }
+    
+    func validatePushNotificationSource(_ notification: [String: AnyObject]) -> Bool {
+        return false
```

Comment:
Not a big fan of default implementations for protocols (https://betterprogramming.pub/swift-why-you-should-avoid-using-default-implementations-in-protocols-eeffddbed46d), but I can see you're just extending the pattern. And it's a pain to update all the actual implementations that use the protocol.